manifests/i/Iriun/IriunWebcam/2.6.8/Iriun.IriunWebcam.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Iriun.IriunWebcam
+PackageVersion: 2.6.8
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  Scope: machine
```

Comment:
Weird, never seen Scope cause a blocking issue(?)
Machine is correct tho

Reply:
Yes, I learnt this from my previous PRs